pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -92,6 +92,18 @@ def test_groupby_aggregation_mixed_dtype():
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_aggregation_multi_level_column():
```

Comment:
can you add the issue number as a comment

Reply:
added issue number and modified test to fail on 0.25.x (verified on 0.25.3)...